src/pages/library/DataBrowserPreview.js

```diff
@@ -52,7 +52,20 @@ const DataBrowserPreview = ({ id }) => {
   const [viewJSON, setViewJSON] = useState()
 
   const tableMap = _.keyBy('name', tables)
-  const tableNames = _.map('name', tables)
+  const selectOptions = _.flow(
+    _.partition(table => table.rowCount > 0),
+    _.toPairs,
+    _.flatMap(([rowIndex, tables]) => {
+      const sortedTables = _.flow(
+        _.sortBy('name'),
+        _.map(({ name, rowCount }) => { return { value: name, rowCount } })
```

Comment:
Could be simplified a bit more since it's a one-liner:

```suggestion
        _.map(({ name, rowCount }) => ({ value: name, rowCount }))
```

Reply:
Oh neat, I didn't know you culd do that, thank you!